cradle-cassandra/src/test/java/com/exactpro/cradle/cassandra/integration/messages/MessageBatchIteratorProviderTest.java

```diff
@@ -85,11 +101,24 @@ public int hashCode() {
         }
     }
 
-    private void setUpOperators() {
+    private void setUpOperators() throws IOException, InterruptedException {
         CassandraDataMapper dataMapper = new CassandraDataMapperBuilder(session).build();
         operators = new CassandraOperators(dataMapper, CassandraCradleHelper.getInstance().getStorageSettings());
     }
 
+
+    private MessageToStore generateMessage (String sessionAlias, Direction direction, int minutesFromStart, long sequence) throws CradleStorageException {
+        return MessageToStore.builder()
+                .bookId(bookId)
+                .sessionAlias(sessionAlias)
+                .direction(direction)
+                .timestamp(dataStart.plus(minutesFromStart, ChronoUnit.MINUTES))
+                .sequence(sequence)
+                .content(CONTENT.getBytes(StandardCharsets.UTF_8))
+                .protocol(protocol)
+                .build();
+    }
+
```

Comment:
It is the same code as in the cradle-cassandra/src/test/java/com/exactpro/cradle/cassandra/integration/messages/GroupedMessageIteratorProviderTest.java